### ISSUE TITLE
Spot instance support

### ISSUE THREAD

**Issue by fourpartswater** [dask/dask-ec2#28]
Is/will there be spot instance support?

**Comment by danielfrg** [comment]
Right now there spot instances are not supported. This is something we could do in the future.

**Comment by stumitchell** [comment]
Initial pr is in https://github.com/dask/dask-ec2/pull/66 it works in practice but is failing tests :-(

**Comment by xxlatgh** [comment]
are there plan to continue the work of supporting spot instance?

**Comment by swamidass** [comment]
Any progress on this?

**Comment by rohan-gt** [comment]
Any update on this?

**Comment by quasiben** [comment]
No, dask-ec2 is largely unsupported at this point.  I'd recommend looking at [dask-cloudprovider](https://cloudprovider.dask.org/en/latest/) or the [dask-helm chart](https://helm.dask.org/) or [dask-kubernetes](https://kubernetes.dask.org/en/latest/) for k8s support.  

You might also consider https://www.saturncloud.io/ or https://coiled.io/ for supported AWS dask deployments
